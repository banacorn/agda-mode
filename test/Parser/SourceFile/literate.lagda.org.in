This line is ordinary text, which is ignored by Agda.

{! 0 !}

#+begin_src agda2
module Whatever where
-- Agda code goes here

{! 1 !}

#+end_src

{! 2 !}

Another non-code line.