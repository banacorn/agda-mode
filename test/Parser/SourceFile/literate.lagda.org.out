Hole [0] (128, 135) => (128, 135) "{! 1 !}"